### PR TITLE
Created feed endpoint to get all articles and gifs together and retur…

### PR DESCRIPTION
**Pull request by mjmandelah07**
…n in order of created

### CODE REVIEW

File: controller/feed-controller.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    // map the feed items
+    const feedData = result.rows.map((row) => ({
+      id: row.id,
+      createdOn: row.created_on,
+      title: row.title,
+      "article/url": row["article/url"],
+      authorId: row.user_id,
+      category: row.category
+    }));
+    res.status(STATUSCODE.OK).json(successResponse(STATUS.Success, feedData));
```
Comment (kelvinator07) [inline controller/feed-controller.js]: Get All Articles DESC, Get All Gifs DESC, Put In One Array, Then Sort By Date

Comment (mjmandelah07) [reply controller/feed-controller.js]: i dont understand

File: controller/feed-controller.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    // Map the combined items
+    const feedData = combinedItems.map((item) => ({
+      id: item.id,
+      createdOn: item.created_on,
+      title: item.title,
+      "article/url": item.article || item.url,
+      authorId: item.user_id,
+      category: item.category,
+    }));
```
Comment (kelvinator07) [inline controller/feed-controller.js]: Don't Map The CombinedIteams, Just Return As It Is

File: controller/feed-controller.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    SELECT *
+    FROM ${tableName}
+    ORDER BY created_on DESC;
+  `;
+  const result = await db.query(query);
+  return result.rows;
+};
+
+// Function to combine and sort items by date
+const combineAndSortItems = (items) => {
```
Comment (kelvinator07) [inline controller/feed-controller.js:21]:
```suggestion
const sortItems = (items) => {
```
This Is Only Sorting It

**Pull request merged**